File: Atividade 1/1.3/tarefa.py
def eh_primo(number):
	for i in range(2,number):
		if(number % i == 0):
			return False
	return True


number = int(input("Digite um numero: "))
print("Numero digitado: {}.".format(number))
print("Numero digitado eh primo? {}.".format(eh_primo(number)))

File: Atividade 1/1.3/Tarefa.java
import java.util.Scanner;

public class Tarefa {
	public static void main(String[] args) {
		long numero;
		Scanner myObj = new Scanner(System.in);

		System.out.print("Digite um numero: ");
		numero = myObj.nextLong();

		System.out.println("Numero digitado: " + numero);
		System.out.println("Eh primo? " + primo(numero));
		myObj.close();
	}

	public static boolean primo(final long numero) {
		if(numero%2 == 0) return false;

		double raiz = Math.sqrt(numero);

		for(long i=3; i<=raiz; i+=2)
			if(numero%i==0) return false;

		return true;
	}
}
